### ISSUE TITLE
Still something wrong with SourceLink

### ISSUE THREAD

**Issue by TheAngryByrd** [demystifyfp/FsToolkit.ErrorHandling#45]
After downloading the [1.1.1 nuget package](https://www.nuget.org/packages/FsToolkit.ErrorHandling), extracting and running `sourcelink print-json`, the URL is still incorrect.

```
 sourcelink print-json FsToolkit.ErrorHandling.TaskResult.pdb 
{"documents":{"/Users/tamizhvendan/DemystifyFpWorks/FsToolkit.ErrorHandling/*":"https://demystifyfp/raw/demystifyfp/FsToolkit.ErrorHandling/1c752129694c21842af13da041ec4def9a152327/*"}}
```

However when building locally against my own I seem to be a sane URL

```
sourcelink print-json FsToolkit.ErrorHandling.pdb
{"documents":{"/Users/jimmybyrd/Documents/GitHub/FsToolkit.ErrorHandling/*":"https://raw.githubusercontent.com/TheAngryByrd/FsToolkit.ErrorHandling/ac855aced4ee0a31119b3aa348021ae28321ac70/*"}}
```

I'm guessing https://github.com/dotnet/sourcelink/issues/278 might be related.

So my remotes listed are using https.

```
git remote -v
origin	https://github.com/TheAngryByrd/FsToolkit.ErrorHandling.git (fetch)
origin	https://github.com/TheAngryByrd/FsToolkit.ErrorHandling.git (push)
upstream	https://github.com/demystifyfp/FsToolkit.ErrorHandling.git (fetch)
upstream	https://github.com/demystifyfp/FsToolkit.ErrorHandling.git (push)
```

@demystifyfp Do yours happen to be using git or are they https?

**Comment by demystifyfp** [comment]
Mine is using `git`.

```
git remote -v
origin	git@demystifyfp:demystifyfp/FsToolkit.ErrorHandling.git (fetch)
origin	git@demystifyfp:demystifyfp/FsToolkit.ErrorHandling.git (push)
```

**Comment by TheAngryByrd** [comment]
It seems they have a bug with git style remotes. 😞. I'll open an issue over there.

**Comment by demystifyfp** [comment]
Updated my gitconfig and published the new version (1.2.1). 

Can you verify is it still there?

**Comment by TheAngryByrd** [comment]
```
sourcelink print-urls FsToolkit.ErrorHandling.pdb
108c97e71af54a0fb3d40544095ae9a5 md5 fsharp /Users/tamizhvendan/DemystifyFpWorks/FsToolkit.ErrorHandling/src/FsToolkit.ErrorHandling/List.fs
https://raw.githubusercontent.com/demystifyfp/FsToolkit.ErrorHandling/90ce9418852b5c41051a10970b450e3e02d3e91e/src/FsToolkit.ErrorHandling/List.fs
```

```
sourcelink print-json FsToolkit.ErrorHandling.pdb
{"documents":{"/Users/tamizhvendan/DemystifyFpWorks/FsToolkit.ErrorHandling/*":"https://raw.githubusercontent.com/demystifyfp/FsToolkit.ErrorHandling/90ce9418852b5c41051a10970b450e3e02d3e91e/*"}}
```

Looks like it's good to me!
